source/draw/gpu/opengl/AndroidContext.ooc

```diff
@@ -116,25 +116,24 @@ AndroidContext: class extends OpenGLContext {
 			super(source, target)
 		result
 	}
-	toRasterAsync: func ~monochrome (gpuImage: OpenGLMonochrome) -> (RasterImage, GpuFence) {
+	toRasterAsync: func ~monochrome (gpuImage: OpenGLMonochrome) -> ToRasterFuture {
 		(buffer, fence) := this toBuffer(gpuImage, this _packMonochrome)
-		(RasterMonochrome new(buffer, gpuImage size), fence)
+		_FenceToRasterFuture new(RasterMonochrome new(buffer, gpuImage size), fence)
 	}
-	toRasterAsync: func ~uv (gpuImage: OpenGLUv) -> (RasterImage, GpuFence) {
+	toRasterAsync: func ~uv (gpuImage: OpenGLUv) -> ToRasterFuture {
 		(buffer, fence) := this toBuffer(gpuImage, this _packUv)
-		(RasterUv new(buffer, gpuImage size), fence)
+		_FenceToRasterFuture new(RasterUv new(buffer, gpuImage size), fence)
```

Comment:
Even `getResult` actually calls the correct function (i.e. `getResult` in the implemented type), it's just the returned value that's corrupt.